server/src/api/v1.py

```diff
@@ -146,6 +146,47 @@ def job_get_id(job_id):
     return job_data
 
 
+@v1.get("/job/search")
+@v1.input(schemas.JobSearchRequest, location="query")
+@v1.output(schemas.JobSearchResponse)
+def search_jobs(query_data):
+    """Search for jobs by tags"""
+    tags = query_data.get("tags")
+    match = request.args.get("match", "any")
+    states = request.args.getlist("state")
```

Comment:
Oh reading the docs I didn't know that you can pass multiple states. Perhaps it would be useful to mention that there